api/src/main/java/javax/enterprise/inject/spi/BeanManager.java

```diff
@@ -229,6 +231,41 @@
     public void fireEvent(Object event, Annotation... qualifiers);
 
     /**
+     * Fire an asynchronous event and notify observers.
```

Comment:
I see. Perhaps having a `BeanManager.event()` or `getEvent()` method that returns an `Event<Object>` object could prevent API swell up?